### DISCUSSION TITLE
How to selectively combine changes in a merge conflict

### DISCUSSION THREAD

**Discussion by jordan-yee** [jesseduffield/lazygit#1431] [Q&A] [answered]
When resolving merge conflicts, the interface seems very intuitive if you want to resolve the conflict via:

1. Keep only the changes from the source branch
2. Keep only the changes from the target branch
3. Keep both changes

What isn't so obvious to me is how to resolve by selectively combining parts of both changes. Is there a built in or recommended way to do this?

**Comment by jesseduffield** [comment, marked as answer]
Unfortunately we don't have support for more granular editing of files. At the moment I typically open the file in my editor to make fine-tuned changes
